core/src/main/java/org/elasticsearch/bootstrap/BootstrapInfo.java

```diff
@@ -120,4 +120,8 @@ public Object remove(Object key) {
         }
         return SYSTEM_PROPERTIES;
     }
+
+    public static void init() {
```

Comment:
It is needed. It forces the class initializer for `BootstrapInfo` to run *before* the security manager is installed; there are invocations in `BootstrapInfo` that will run into `AccessDeniedException` if executed after the security manager is installed.